.circleci/config.yml

```diff
@@ -36,7 +36,10 @@ jobs:
           at: ~/repo
 
       - run: npm run linter:check
-      - run: npm test
+      - run:
+          name: Tests
+          no_output_timeout: 15m # a workaround until we make it faster later
```

Comment:
Ohhhh I see now, so it was a circle ci timeout. How weird. Happy to not spend any more time on this as we will move to GH actions later